tools/xtce_generation/telemetry_generator.py

```diff
@@ -245,7 +269,9 @@ def create_container_set(
             The updated TelemetryMetaData element.
         """
         # Create ContainerSet element
-        container_set = Et.SubElement(telemetry_metadata, "xtce:ContainerSet")
+        container_set = Et.SubElement(telemetry_metadata, "xtce:ContainerSet")  # type: ignore[arg-type]
+        # Argument 1 to "SubElement" has incompatible type
+        # "dict[Any, Any]"; expected "Element"
```

Comment:
```suggestion
        container_set = Et.SubElement(telemetry_metadata, "xtce:ContainerSet")
```

Should be able to remove these with the above fix